core/src/main/java/com/datastax/oss/driver/api/core/AsyncPagingIterable.java

```diff
@@ -101,7 +100,7 @@ default ElementT one() {
    * <p>Note that both instances share the same underlying data: consuming elements from the
    * transformed iterable will also consume them from this object, and vice-versa.
    */
-  default <TargetT> AsyncPagingIterable<TargetT> map(
+  default <TargetT> MappedAsyncPagingIterable<TargetT> map(
```

Comment:
`AsyncPagingIterableWrapper<SourceT, TargetT>` is an internal class.
`MappedAsyncPagingIterable<TargetT>` is how it's exposed in the public API.

It has a different name because the fact that it's a wrapper is an implementation detail, the only thing that matters to clients is that the elements have been transformed to something that is not a `Row`. That's also why it doesn't have the `SourceT` type parameter, calling code can't do anything with it so it's not relevant.